Usage test failed.
```
Severity	Code	Description	Project	File	Line	Suppression State	Details
Error		CMake Error at F:\testspace\tuse\CMakeLists.txt:16 (find_package):
  Could not find a package configuration file provided by "usearch" with any
  of the following names:

    usearchConfig.cmake
    usearch-config.cmake

  Add the installation prefix of "usearch" to CMAKE_PREFIX_PATH or set
  "usearch_DIR" to a directory containing one of the above files.  If
  "usearch" provides a separate development package or SDK, be sure it has
  been installed.	tuse	F:\testspace\tuse\CMakeLists.txt	16	
```

--- a/ports/usearch/vcpkg.json
+++ b/ports/usearch/vcpkg.json
@@ -15,9 +15,6 @@
       "host": true
     }
   ],
-  "default-features": [
-    "fp16"
-  ],
   "features": {
     "fp16": {
       "description": "Use software emulation for half-precision types",

--- a/versions/u-/usearch.json
+++ b/versions/u-/usearch.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "f23392616f3aa730c431737dcdf774551dac5737",
+      "git-tree": "50da7c77dca04b357388a72af5e3a91c5176dfb0",
       "version": "2.17.2",
       "port-version": 1
     },

